docker-compose.yml

```diff
@@ -113,26 +75,71 @@ services:
       - AWS_ACCESS_KEY_ID=.
       - AWS_SECRET_ACCESS_KEY=.
     depends_on:
-      - ipfs
+      - cas_ipfs
       - database
       - localstack
       - cas_api
       - launch_contract
       - ganache
       - cas
-    volumes:
-      - './migrations:/cas/migrations'
```

Comment:
We don't need to run the migrations anymore?

Reply:
The cas_api should run the migrations automatically

Reply:
Oh wait, you're right. Ignore my comment.